### ISSUE TITLE
Lazy load not working with the latest master source code.

### ISSUE THREAD

**Issue by Smilefounder** [MarkPieszak/aspnetcore-angular2-universal#526]
Hello team,

Lazy load feature doesn't work, could you please help double check?

Thanks,
Huy

**Comment by Adondriel** [comment]
Working for me, on my master branch, on my fork... Could you give more info?
https://github.com/Adondriel/aspnetcore-angular2-universal for my version.

**Comment by Smilefounder** [comment]
@Adondriel thanks for your reply, 5 days ago I tried to get the latest master source code (with Angular 4), compiled and run but the lazy module keeps loading at the page load. Let me try again with the latest master source code then keep you update.

**Comment by Smilefounder** [comment]
@Adondriel : still got the same issue with the latest master code just now.

**Comment by Adondriel** [comment]
Hmm, what versions of everything are you running? That would help out, gotta make sure all the dev tools are up to date. The other thing is, I 100% suggest using VS 2017 now, as it is actually pretty good at launching this stuff.

Are you getting any errors? What does your network log look like? How are you starting the project? It sounds like you're running the frontend but, without the backend somehow.

**Comment by Gandii** [comment]
Think im getting the same issue. No fancy changes from current master, just clone and run it. Happens both with prod and dev.

`vendor.js:sourcemap:1789 ERROR Error: Uncaught (in promise): Error: Cannot find module './containers/lazy/lazy.module.ngfactory'.
Error: Cannot find module './containers/lazy/lazy.module.ngfactory'.
    at vendor.js:sourcemap:49324
    at ZoneDelegate.invoke (vendor.js:sourcemap:95910)
    at Object.onInvoke (vendor.js:sourcemap:5090)`

**Comment by Gandii** [comment]
Sorry, ignore my post. Seems to be linked to running build:p when in dev mode and/or vise versa.
I also missed preloadingStrategy beeing set too PreloadAllModules
